docs/slack.rst

```diff
@@ -161,13 +161,26 @@ Creating channels
 
 If you didn't see a channel that matches your interests, make sure to browse the complete channel list in Slack.
 
-If you still can't find a suitable channel, you can propose a new channel. To propose a channel:
-
-1. Find two other people to cosponsor your channel idea. Ask in related channels to find people who share your interest.
-2. Together, come up with a channel name and write a channel description.
-3. Post in **#meta** requesting a channel. Be sure to share your proposed channel name, description, and to tag your cosponsors.
-
-The Write the Docs moderation team will review your proposal.
+If you still can't find a suitable channel, you can propose a new channel.
+We use this process to help channels succeed at creating discussion and introducing documentarians to others who share their interests.
+To propose a channel:
+
+#. Find two other people to cosponsor your channel idea.
+   Ask in related channels to find people who share your interest.
+#. Together, come up with a channel name and write a channel description.
+#. Post in **#meta** requesting a channel.
+   Be sure to share your proposed channel name, description, and to tag your cosponsors.
+#. Start thinking about how to seed the new channel with discussion, with an open-ended question, a link to discuss, or a hot take.
+
+The Write the Docs moderation team will review your proposal to make sure the new channel will be active, durable, and won't duplicate or create confusion with another channel.
+Whenever possible, the moderators work with channel sponsors on improvements to the proposal.
```

Comment:
@janine-c what do you think of the revisions here? Following up on https://github.com/writethedocs/www/commit/be680b0c5f8eac050b9ab8b53c4259c19674abec#r122875538

Reply:
Love it! I wonder if "durable" will confuse people - maybe "sustainable" might make more sense? Are we thinking about making sure that the channel will stay active/useful over time here?

Reply:
Yes, I could see that might be confusing. I was trying to capture the idea of "not temporary" (e.g., I don't want people suggesting channels that are for one-off events). A couple of alternatives might be:

- Long-lived
- Permanent
- Enduring

Or I could just omit it and go with "…active and won't duplicate…".

Any thoughts?

Reply:
Yeah, maybe we could say something like "...the new channel will remain active and won't duplicate..."? I think that even when channels feel like they'll probably meet that criteria, we can't know if they'll be permanent, but I think that phrasing hits on the idea that we don't want to have a channel that's, say, only related to a single year's hackathon or anything like that.

Reply:
"Remain active" I like it, done! ✅